SPECS/cloud-hypervisor/cloud-hypervisor.spec

```diff
@@ -71,6 +72,9 @@ Cloud Hypervisor is an open source Virtual Machine Monitor (VMM) that runs on to
 %setup -q -n %{name}-%{version}
 %if 0%{?using_vendored_crates}
 tar xf %{SOURCE1}
+pushd vendor/libz-sys/src/zlib
```

Comment:
I don't think this is actually a dependency.  The build logs do not show zlib being compiled or libz being linked.

I think we might want to remove this from the vendor tarball though.

Reply:
This was included in the build's test infrastructure. Per Rohit:

_It turns out that cloud-hypervisor's dependency on libz is like this: test_infra -> ssh2 -> libssh2-sys -> libz-sys 
 
test_infra is part of cloud-hypervisor's build dependency [[link](https://github.com/cloud-hypervisor/cloud-hypervisor/blob/42ca292dce4292f0f69bd7cecd7db09eee3b64bb/Cargo.toml#L91C18-L91C18)] which needs ssh2 [[link](https://github.com/cloud-hypervisor/cloud-hypervisor/blob/42ca292dce4292f0f69bd7cecd7db09eee3b64bb/test_infra/Cargo.toml#L14C1-L14C1)]_